__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -97,4 +97,92 @@ describe('TicketsTable', () => {
     render(<TicketsTable items={[]} />);
     expect(screen.getByText('Urgent')).toBeInTheDocument();
   });
+
+  it('renders Urgent on table if a case is Urgent', () => {
+    render(<TicketsTable items={[]} />);
+    expect(screen.getByText('Urgent')).toBeInTheDocument();
+  });
+
+  it('should sort the table when column headers are clicked', () => {
```

Comment:
It seems like multiple things are being tested in this function. It might make sense to separate them into different functions so that it's easier to isolate the problem when something breaks